Procfile

```diff
@@ -1,2 +1,3 @@
+release: bundle exec rake db:migrate
 web: bin/start-pgbouncer bundle exec puma -C config/puma.rb
-worker: bundle exec rake jobs:work
\ No newline at end of file
+worker: bundle exec rake jobs:work
```

Comment:
since we're using Rails 5+ we should probably replace both with `bundle exec rails db:migrate` instead of using rake. I'm honestly not sure why both work but I've switched to `rails db:migrate` locally a long time ago :D